turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksSession.java

```diff
@@ -141,6 +141,10 @@ public void onReceiveValue(String s) {
             public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request) {
                 String newLocation = request.getUrl().toString();
                 TurbolinksLog.d("shouldOverrideUrlLoading " + view.getUrl() + " new: " + newLocation);
+                if (turbolinksAdapter.shouldOverrideRequest(request)) {
```

Comment:
How does Wealthbox signal to the native app that this behavior should happen?

Reply:
By Wealthbox do you mean the web app? It doesn't do anything in particular, but this callback for the turbolinks webview allows you to catch a request and either allow it or cancel it and do something else. So we cancel it in the case of an interstitial.

Reply:
https://github.com/starburstlabs/crm-turbolinks-android/blob/ea281f3c340ca16f9fbd7b5634c883604d81f039/app/src/main/java/com/starburstlabs/wealthbox_android/utils/TurbolinksInterface.kt#L413-L421